packages/webapp/pages/[userId]/index.tsx

```diff
@@ -85,24 +78,30 @@ const readHistoryToTooltip = (
   );
 };
 
+interface RankHistoryProps {
+  rank: number;
+  rankName: string;
+  count: number;
+}
```

Comment:
Maybe we should also extract this from the `lib/ranks` props?
then we make sure they are the same type if we ever change something.

Reply:
Good point!